main.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env ruby
+
+# Create methods that do the following:
+# List all books.
+# List all people.
+# Create a person (teacher or student, not a plain Person).
+# Create a book.
+# Create a rental.
+# List all rentals for a given person id.
+
+require './classroom'
+require './person'
+require './teacher'
+require './student'
+
+def list_books
+  books = [Book.new('Harry Potter', 'JK Rowling'),
+           Book.new('Lord of the Rings', 'JRR Tolkien'),
+           Book.new('The Hobbit', 'JRR Tolkien')]
+
+  books.each do |book|
+    puts book.title
+  end
+end
+
+def list_people
+  people = [Teacher.new('Pysics', 20, 'Harry Maguire', parent_permission: false),
+            Student.new('Pysics', 20, 'Jon Snow', parent_permission: false)]
+
+  people.each do |person|
+    puts person.name
+  end
+end
```

Comment:
- [ ] This method is meant to list all people that were created by a user. However, you are creating the people automatically by yourself here which is not what this method is meant to achieve. Kindly remove these people you created and list only the people that were created by a user.